polars/polars-core/src/fmt.rs

```diff
@@ -902,6 +902,24 @@ Series: 'foo' [i32]
 	97
 	98
 	99
+]"#,
+            format!("{:?}", s)
+        );
+    }
+
+    #[test]
```

Comment:
Can you hoist this test to the python library? We want to reduce compilation times of the rust tests.

Reply:
Ahh yes sure thing.

Reply:
@ritchie46 I think we are good now if this looks ok to you.